### PR TITLE
materials project properties

### PR DESCRIPTION
**Pull request by kjappelbaum**
partially addressing #424

### CODE REVIEW

**Comment by kjappelbaum** [comment]
skipping on the total magnetization (https://huggingface.co/datasets/kjappelbaum/chemnlp-mp-magnetization) as, when reading up on it, it seems not super consistent

File: data/tabular/mp_anisotropy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        title = {Charting the complete elastic properties of inorganic crystalline compounds},
+        journal = {Sci Data}
+      }
+templates:
+    - The {elastic_anisotropy__names__noun} of {#the compound|the solid|!} {formula#} is {elastic_anisotropy#}.
+    - |-
+      Question: How large is the {elastic_anisotropy__names__noun} of {#the compound|the solid|!} {formula#}?
+      Answer: The {elastic_anisotropy__names__noun} of {#the compound|the solid|!} {formula#} is {elastic_anisotropy#}.
+    - |-
+      User: {#I would like to|I want tp!} know the {elastic_anisotropy__names__noun} of {#the compound|the solid|!} {formula#}.
```
Comment (MicPie) [inline data/tabular/mp_anisotropy/meta.yaml:86]:
```suggestion
      User: {#I would like to|I want to!} know the {elastic_anisotropy__names__noun} of {#the compound|the solid|!} {formula#}.
```

File: data/tabular/mp_shear_modulus/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        title = {Charting the complete elastic properties of inorganic crystalline compounds},
+        journal = {Sci Data}
+      }
+templates:
+    - The {shear_modulus__names__noun} of {#the compound|the solid|!} {formula#} is {shear_modulus#} {shear_modulus__units}.
+    - |-
+      Question: How large is the {shear_modulus__names__noun} of {#the compound|the solid|!} {formula#}?
+      Answer: The {shear_modulus__names__noun} of {#the compound|the solid|!} {formula#} is {shear_modulus#} {shear_modulus__units}.
+    - |-
+      User: {#I would like to|I want tp!} know the {shear_modulus__names__noun} of {#the compound|the solid|!} {formula#}.
```
Comment (MicPie) [inline data/tabular/mp_shear_modulus/meta.yaml:86]:
```suggestion
      User: {#I would like to|I want to!} know the {shear_modulus__names__noun} of {#the compound|the solid|!} {formula#}.
```

File: data/tabular/mp_bulk_modulus/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        title = {Charting the complete elastic properties of inorganic crystalline compounds},
+        journal = {Sci Data}
+      }
+templates:
+    - The {bulk_modulus__names__noun} of {#the compound|the solid|!} {formula#} is {bulk_modulus#} {bulk_modulus__units}.
+    - |-
+      Question: How large is the {bulk_modulus__names__noun} of {#the compound|the solid|!} {formula#}?
+      Answer: The {bulk_modulus__names__noun} of {#the compound|the solid|!} {formula#} is {bulk_modulus#} {bulk_modulus__units}.
+    - |-
+      User: {#I would like to|I want tp!} know the {bulk_modulus__names__noun} of {#the compound|the solid|!} {formula#}.
```
Comment (MicPie) [inline data/tabular/mp_bulk_modulus/meta.yaml:86]:
```suggestion
      User: {#I would like to|I want to!} know the {bulk_modulus__names__noun} of {#the compound|the solid|!} {formula#}.
```

**Review by MicPie — CHANGES_REQUESTED** [review]
Just one type that got copied over to the yamls.
When this if fixed we can merge! :-)

**Review by MicPie — APPROVED** [review]
lgtm

**Pull request merged**